Sources/TokamakCore/MountedViews/MountedApp.swift

```diff
@@ -48,7 +47,7 @@ final class MountedApp<R: Renderer>: MountedCompositeElement<R> {
     reconciler.reconcile(
       self,
       with: element,
-      getElementType: { $0.type },
+      getElementType: { type(of: $0) },
```

Comment:
Could this be?

```suggestion
      getElementType: type(of:),
```

Still a bit of a swift beginner